uwblueprint/marillac-place
# Edit Residents Modal

## Notion ticket link

[Edit Residents Modal](https://www.notion.so/uwblueprintexecs/Edit-Residents-Modal-73c2ed32aea148488ee0fea30185047a?pvs=4)



## Implementation description
* Add residents modal element
* Modify mock data 
* Add other ways for Form field to display error colouring
* Make it so only the Cancel button can close a modal 



## Steps to test
1. Go to residents page
2. Click the pencil button
3. Change residentId to a residentId in a different row. Pressing submit should show an error
4. Change roomNumber to a roomNumber in a different row. Pressing submit should show an error
5. Unable to input a non number into residentId and roomNumber
6. Leaving a required field blank will also display boxes as red



## What should reviewers focus on?
* 


## Checklist
